ContractExamples/scripts/prepare-timelock.sh

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env bash
+#
+# Deploys a token contract, initializes it and mints tokens, then deploys a timelock contract adn deposits
+#
+# Jure Kukovec, 2024
+#
+# @license
+# [Apache-2.0](https://github.com/freespek/solarkraft/blob/main/LICENSE)
+
+set -e
+
+SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+RPC="https://soroban-testnet.stellar.org:443"
+
+ALICE=alice
+soroban keys address $ALICE || (echo "add the account $ALICE via soroban keys generate"; exit 1)
+BOB=bob
+soroban keys address $BOB || (echo "add the account $BOB via soroban keys generate"; exit 1)
+
+set -x
+
+TOKEN="`
+    soroban contract deploy \
+    --source-account alice \
+    --network testnet\
+    --rpc-url $RPC\
+    --network-passphrase "Test SDF Network ; September 2015"\
+    --wasm $SCRIPT_DIR/../contracts/token/target/wasm32-unknown-unknown/release/soroban_token_contract.wasm\
+    `";
```

Comment:
I don't see what the other double quotes give us here?
In any case, I wouldn't nest double quotes.

```suggestion
TOKEN=$(
    soroban contract deploy \
    --source-account alice \
    --network testnet\
    --rpc-url $RPC\
    --network-passphrase "Test SDF Network ; September 2015"\
    --wasm $SCRIPT_DIR/../contracts/token/target/wasm32-unknown-unknown/release/soroban_token_contract.wasm\
    )
```